scripts/sign_with_gpg.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
Can you make the existing `scripts/release/publish.sh` script call this script to do it's signing work?